docs/tutorial.rst

```diff
@@ -736,39 +720,31 @@ Also note that recombinations will still occur in the gaps between chromosomes,
 with corresponding trees in the tree sequence. This will be fixed in a future
 release.
 
--------------------------------------
+.. _sec_tutorial_hybrid_simulations:
+
+******************
 Hybrid simulations
--------------------------------------
+******************
 
-In some situations Wright-Fisher simulations are desireable yet less
+In some situations Wright-Fisher simulations are desireable but less
 computationally efficient than coalescent simulations, for example simulating a
 small sample in a recently admixed population. In these cases, a hybrid model
 offers an excellent tradeoff between simulation accuracy and performance.
 
-This is done through a SimulationModelChange event, which is a special type of
-demographic event. 
+This is done through a :class:`.SimulationModelChange` event, which is a special type of
+demographic event.
 
 For example, here we switch from the discrete-time Wright-Fisher model to the
 standard Hudson coalescent 100 generations in the past:
 
-.. code-block:: python
-
-    num_wf_generations = 100
+.. todo:: FIXME this doesn't work yet.
```

Comment:
This will work once this PR is merged though right?

Reply:
Yes, this was put in earlier so I didn't forget about it. I'll clear it up for the final version.